tests/cases/unittests/session.ts

```diff
@@ -1,5 +1,7 @@
 /// <reference path="..\..\..\src\harness\harness.ts" />
 
+var expect: typeof _chai.expect = _chai.expect;

```

Comment:
I expect this was moved from [here](https://github.com/Microsoft/TypeScript/pull/6140/files#diff-13d9171803710d040ef4b5b12950a65aL34) to here because the lint rule couldn't handle the global being declared in another file, as lints are done per-file, so within the file it was originally declared in it was unused.